src/clients/ca.h

```diff
@@ -2,17 +2,14 @@
 // Licensed under the Apache 2.0 License.
 #pragma once
 
-#include "crypto/mbedtls/error_string.h"
-#include "crypto/mbedtls/mbedtls_wrappers.h"
+#include "crypto/openssl/openssl_wrappers.h"
 #include "crypto/pem.h"
 #include "ds/buffer.h"
 
 #include <exception>
 
-// This is a copy of src/tls/ca.h
-// Moving tls to OpenSSL means this different client implementation needs
-// to be isolated while we change tls_endpoint. Once that's done, we can
-// come back here and refactor this too.
+using namespace crypto;
```

Comment:
I didn't know you copied this over, I thought it's just old stuff. I'll give it a try, although I prefer the one currently in src/clients because it uses the unique wrapper objects.

Reply:
All done already, there's a small change for a CI failure if I overlooked something.